client/lib/signup/step-actions.js

```diff
@@ -169,9 +169,10 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 	const flowToCheck = flowName || lastKnownFlow;
 
 	if (
-		flowToCheck === 'onboarding' &&
-		newSiteParams.options.theme !== defaultSiteTypeTheme &&
-		abtest( 'verticalSuggestedThemes' ) === 'test'
+		get( getSignupDependencyStore( state ), 'useThemeHeadstart' ) ||
```

Comment:
If the theme step is present in the flow, we should default to theme annotations.